definitions/features/pulp3.rb

```diff
@@ -0,0 +1,22 @@
+require 'foreman_maintain/utils/service/systemd'
+
+class Features::Pulp3 < ForemanMaintain::Feature
+  metadata do
+    label :pulp3
+
+    confine do
+      ForemanMaintain::Utils::Service::Systemd.new('pulpcore-api', 0).exist?
+    end
+  end
+
+  def services
+    [
+      system_service('pulpcore-api', 10),
+      system_service('pulpcore-content', 10),
+      system_service('pulpcore-resource-manager', 10),
+      system_service('pulpcore-worker@*', 20, :all => true),
+      system_service('redis', 30),
+      system_service('httpd', 30)
```

Comment:
@jlsherrill what do you think about the order here? Do we want to start and stop services in a particular order?

This is the order with the current values:

```
| displaying rh-mongodb34-mongod                   [OK]                         
| displaying postgresql                            [OK]                         
| displaying pulpcore-api                          [OK]                         
/ displaying pulpcore-content                      [OK]                         
/ displaying pulpcore-resource-manager             [OK]                         
/ displaying qdrouterd                             [OK]                         
/ displaying qpidd                                 [OK]                         
/ displaying squid                                 [OK]                         
/ displaying pulp_celerybeat                       [OK]                         
/ displaying pulp_resource_manager                 [OK]                         
/ displaying pulp_streamer                         [OK]                         
/ displaying pulp_workers                          [OK]                         
- displaying pulpcore-worker@*                     [OK]                         
- displaying tomcat                                [OK]                         
- displaying httpd                                 [OK]                         
- displaying puppetserver                          [OK]                         
- displaying redis                                 [OK]                         
- displaying foreman-proxy                         [OK]        
```

Reply:
We might ask the pulp team if it matters, i might could see it being better to shut down the api, then the resource-manager and then the workers, but in practice it may not matter.   I'd reach out to the pulp team and ask if they have a preference.

Reply:
also, does the order reverse itself when starting vs stopping?

Reply:
It does reverse